app/templates/merchant/view_listings.html

```diff
@@ -24,13 +49,16 @@
                 <input type="checkbox" name="favorite" tabindex="0" {{ checked }}>
                 <label>Only show favorites</label>
             </div>
+            <br>
+
         </div>
 
+
         {% if not name_search_term and not min_price and not max_price%}
             {% set display = 'display:none' %}
         {% endif %}
         <div class="twelve wide field advanced-search" style="{{ display }}">
-            <label>Search company and vendor names</label>
+            <label>Search company names</label>
```

Comment:
The Vendor.first_name and Vendor.last_name did not return any results. I am not sure if this was a model issue or something else.